integration_tests/test_contract.py

```
@@ -85,61 +86,68 @@ async def test_connect_flow(connect_mantra):
     balance = await ERC20.fns.balanceOf(owner).call(w3, to=token)
-    await ERC20.fns.mint(owner, 1000).transact(w3, owner, to=token)
+    await ERC20.fns.mint(owner, 1000).transact(w3, account, to=token)
     assert await ERC20.fns.balanceOf(owner).call(w3, to=token) == balance + amt
 
     # test_weth
     weth = WETH(to=WETH_ADDRESS)
     before = await balance_of(w3, ZERO_ADDRESS, owner)
-    receipt = await weth.fns.deposit().transact(w3, owner, value=1000)
+    receipt = await weth.fns.deposit().transact(w3, account, value=1000)
     fee = receipt["effectiveGasPrice"] * receipt["gasUsed"]
     await balance_of(w3, WETH_ADDRESS, owner) == 1000
```

Ruff (0.12.2)

Pointless comparison. Did you mean to assign a value? Otherwise, prepend `assert` or remove it.

(B015)